worker-service/src/main/java/com/ledger/owner/workerservice/web/WorkerController.java

```diff
@@ -0,0 +1,46 @@
+package com.ledger.owner.workerservice.web;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.common.ledger.utils.LedgerResponse;
+import com.common.ledger.utils.ResponseUtil;
+import com.ledger.owner.workerservice.service.WorkerService;
+
+@RestController
+public class WorkerController {
+
+	WorkerService workerService;
+	
+	/**
+	 * 
+	 * @param wName param value expecting with worker name
+	 * @param wId param value expecting with worker id
+	 * @return
+	 */
+	@ResponseBody ResponseEntity<Map<String, Object>> getWorkerDetails(@RequestParam(name = "workerName", required=false) String wName,
```

Comment:
Better we add ModelObject as a parameter rather than giving saperate parameters

Reply:
Sure we will make as modelObject which we already have.. since i thought to make this api as GET I used as param valuel